tests/python_tests/helpers/unpack.py

```diff
@@ -48,6 +48,9 @@ def bytes_to_bfloat16(byte_list):
         for i in range(0, len(limited_packed_list), 2)
     ]
 
+    for i in range(0, len(result), 2):
+        result[i], result[i + 1] = result[i + 1], result[i]
+
```

Comment:
```suggestion
    if unpack_src == pack_dst:
        for i in range(0, len(result), 2):
            result[i], result[i + 1] = result[i + 1], result[i]

```
Similar to suggestion for `pack.py` you need to add condition in order to prevent reversing bug fix for format sweeps. On line 59 you will see that the same thing is done, consecutive byte pairs are reversed. This was a bug found in `unpack.py` when input format is `Bfp8_b` and the input and output formats are not the same. I suggest setting another flag that checks if our input and output is the same, otherwise when we run this test with a various format combinations the condition on line 59 will be true in some cases, which aims to reverse consecutive byte pairs, however this will already be done by loop at line 51 and will reverse the bug fix, causing the passing test to start failing.